package.json

```diff
@@ -5,8 +5,8 @@
   "main": "bot.js",
   "scripts": {
     "start": "node bot.js",
-    "test": "mocha 'test/**/*.js'",
-    "migrate": "knex migrate:latest"
+    "migrate": "knex migrate:latest",
+    "test": "eslint 'lib/**/*.js' 'skills/**/*.js' && mocha 'test/**/*.js'"
```

Comment:
I recommend having a standalone script, just to run linting (ie. `npm run eslint`).

Reply:
You could have 2 scripts: `npm run mocha` and `npm run eslint`, then the `npm test` script could just run both of those other scripts: `npm run eslint && npm run mocha`.

Reply:
@nrambeck where do you choose those two scripts? Do you write that in the package.json?

Reply:
I did it this way only because on the card it says `npm test should run this command as well`

Reply:
I can 100% change it if we want them to run separately..